runtime/centrifuge/src/lib.rs

```diff
@@ -1116,7 +1041,7 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	IntegrateCollatorSelection<Runtime>,
+	(),
```

Comment:
The runtime upgrade type parameter here has a default of `()`, so this can technically be completely omitted. Not gonna block on it, though.